src/ploneintranet/suite/setuphandlers.py

```diff
@@ -109,6 +113,37 @@ def testing(context):
             users.append(user)
     create_users(context, users, 'avatars')
 
+    # We use following fixed tags
+    tags = ['Rain', 'Sun', 'Planes', 'ICT', ]
+
+    # We use fixed dates, we need thes to be relative
+    #publication_date = ['just now', 'next week', 'next year', ]
+
+    # make newsitems
+    news_content = [
+        {'title': 'Second Indian Airline to join Global Airline Alliance',
+         'description': 'Weak network in growing Indian aviation market',
+         'tags': [tags[0]],
+         'publication_date': ''},
+
+        {'title': 'BNB and Randomize to codeshare',
+         'description': 'Starting September 10, BNB passengers will be' +
```

Comment:
No + please. Python magically concatenates strings within any form of braces. Just removing the + will still work